envoyauth/request.go

```diff
@@ -56,6 +67,13 @@ func RequestToInput(req any, logger logging.Logger, protoSet *protoregistry.File
 		body = req.GetAttributes().GetRequest().GetHttp().GetBody()
 		headers = req.GetAttributes().GetRequest().GetHttp().GetHeaders()
 		rawBody = req.GetAttributes().GetRequest().GetHttp().GetRawBody()
+
+		if req.GetAttributes().GetSource().GetPrincipal() != "" {
+			sourcePrincipal, err = ast.InterfaceToValue(req.GetAttributes().GetSource().GetPrincipal())
```

Comment:
Do we need the manual AST-conversion? Doesn't seem required for the other fields.